lib/awful/tag.lua

```diff
@@ -409,6 +409,36 @@ function tag.getgap(t)
     return tag.getproperty(t, "useless_gap") or beautiful.useless_gap or 0
 end
 
+--- Set size fill policy for the master client(s)
+-- @tparam string policy Can be set to
+-- "expand" (fill all the available workarea) or
+-- "mwfact" (fill only an area inside the master width factor)
+-- @tparam tag t The tag to modify, if null tag.selected() is used.
+function tag.setmfpol(enabled, t)
+    local t = t or tag.selected()
+    tag.setproperty(t, "master_fill_policy", enabled)
+end
+
+--- Toggle size fill policy for the master client(s)
+-- @tparam tag t The tag to modify, if null tag.selected() is used.
+function tag.togglemfpol(t)
+    if tag.getmfpol(t) == "expand" then
+        tag.setmfpol("mwfact", t)
+    else
+        tag.setmfpol("expand", t)
+    end
+end
+
+--- Get size fill policy for the master client(s)
+-- @tparam tag t Optional tag.
+-- @treturn string Can be set to
```

Comment:
s/Can bet set to/Possible values are/

And also mention the default of `expand` here.